lib/normalizers.js

```diff
@@ -55,6 +55,30 @@ module.exports.abbreviateDirectionals = function(val) {
 
 };
 
+module.exports.abbreviateStreetSuffexes = function(val) {
+  if (!_.isString(val)) {
+    return val;
+  }
+
+  var terms = {
+    'STREET': 'ST',
+    'AVENUE': 'AVE',
+    'BOULEVARD': 'BLVD',
+    'ROAD': 'RD',
+    'DRIVE': 'DR'
+  };
```

Comment:
See https://github.com/pelias/schema/blob/master/street_suffix.js for a complete list of street types/abbreviations culled from Tiger data and my memory.